WalletWasabi.Daemon/Rpc/WasabiJsonRpcService.cs

```diff
@@ -542,4 +563,60 @@ private static bool TryParseMnemonic(string mnemonicStr, [NotNullWhen(true)] out
 			return false;
 		}
 	}
+
+	private JsonRpcResult GetRpcResultFromSmartCoin(SmartCoin coin, uint serverTipHeight)
+	{
+		return new JsonRpcResult
+		{
+			["txid"] = coin.TransactionId.ToString(),
+			["index"] = coin.Index,
+			["amount"] = coin.Amount.Satoshi,
+			["anonymityScore"] = coin.HdPubKey.AnonymitySet,
+			["confirmed"] = coin.Confirmed,
+			["confirmations"] = coin.Confirmed ? serverTipHeight - (uint)coin.Height.Value + 1 : 0,
+			["label"] = coin.HdPubKey.Labels.ToString(),
+			["keyPath"] = coin.HdPubKey.FullKeyPath.ToString(),
+			["address"] = coin.HdPubKey.GetAddress(Global.Network).ToString(),
+			["excludedFromCoinjoin"] = coin.IsExcludedFromCoinJoin
+		};
+	}
+
+	private JsonRpcResult GetRpcResultFromCoin(Coin coin)
+	{
+		return new JsonRpcResult
+		{
+			["amount"] = coin.Amount,
+			["outpoint"] = coin.Outpoint
+		};
+	}
```

Comment:
You can use arrow functions to reduce code complexity. (Here and in another mapper methods)

```suggestion
	private JsonRpcResult GetRpcResultFromCoin(Coin coin) => new JsonRpcResult
	{
		["amount"] = coin.Amount,
		["outpoint"] = coin.Outpoint
	};
```